### ISSUE TITLE
Fix tags for en-US JS pages

### ISSUE THREAD

**Issue by ddbeck** [mdn/sprints#2785]
I want to know that all JS pages are tagged to their corresponding recipe in stumptown, so I can get complete results for the linting that will come out of #2784.

# Acceptance criteria

- [ ] `scraper-ng` identifies one and only one recipe for every JS page

**Comment by ddbeck** [comment]
Retitled in response to https://github.com/mdn/sprints/issues/2784#issuecomment-582889192.

**Comment by Elchi3** [comment]
Looked at the linter output for this and here's a checklist to burn down for some tagging party.

Non-unique recipes:

1. -  [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Indexed_collections
2. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects
3. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error
4. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/columnNumber
5. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/fileName
6. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/lineNumber
7. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message
8. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name
9. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack
10. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toSource
11. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString
12. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/EvalError/EvalError
13. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/InternalError/InternalError
14. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError/RangeError
15. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError/ReferenceError
16. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError/SyntaxError
17. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError/TypeError
18. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError/URIError
19. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/CompileError
20. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Error.description
21. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Error.number
22. - [ ] https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Error.stackTraceLimit

Missing recipe:

- [ ] 1. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript
- [ ] 2. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/About_JavaScript
- [ ] 3. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Closures
- [ ] 4. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop
- [ ] 5. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Differential_inheritance_in_JavaScript
- [ ] 6. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties
- [ ] 7. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
- [ ] 8. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Index
- [ ] 9. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Introduction_to_using_XPath_in_JavaScript
- [ ] 10. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions
- [ ] 11. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes
- [ ] 12. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Groups_and_Ranges
- [ ] 13. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Quantifiers
- [ ] 14. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Unicode_Property_Escapes
- [ ] 15. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures
- [ ] 16. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Language_Resources
- [ ] 17. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference
- [ ] 18. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/About
- [ ] 19. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes
- [ ] 20. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Class_fields
- [ ] 21. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor
- [ ] 22. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends
- [ ] 23. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
- [ ] 24. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Deprecated_and_obsolete_features
- [ ] 25. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Deprecated_and_obsolete_features/The_legacy_Iterator_protocol
- [ ] 26. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators
- [ ] 27. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators
- [ ] 28. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Array_comprehensions
- [ ] 29. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators
- [ ] 30. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators
- [ ] 31. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator
- [ ] 32. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators
- [ ] 33. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator
- [ ] 34. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
- [ ] 35. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Expression_closures
- [ ] 36. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function
- [ ] 37. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Generator_comprehensions
- [ ] 38. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Grouping
- [ ] 39. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Legacy_generator_function
- [ ] 40. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators
- [ ] 41. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator
- [ ] 42. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer
- [ ] 43. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence
- [ ] 44. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
- [ ] 45. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Pipeline_operator
- [ ] 46. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors
- [ ] 47. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
- [ ] 48. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/async_function
- [ ] 49. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await
- [ ] 50. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/class
- [ ] 51. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete
- [ ] 52. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function*
- [ ] 53. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in
- [ ] 54. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof
- [ ] 55. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new
- [ ] 56. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target
- [ ] 57. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super
- [ ] 58. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this
- [ ] 59. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof
- [ ] 60. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void
- [ ] 61. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/yield
- [ ] 62. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/yield*
- [ ] 63. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
- [ ] 64. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters
- [ ] 65. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions
- [ ] 66. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters
- [ ] 67. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
- [ ] 68. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
- [ ] 69. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set
- [ ] 70. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
- [ ] 71. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar
- [ ] 72. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects
- [ ] 73. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
- [ ] 74. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer
- [ ] 75. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AsyncIterator
- [ ] 76. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics
- [ ] 77. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
- [ ] 78. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
- [ ] 79. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator
- [ ] 80. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity
- [ ] 81. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
- [ ] 82. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Collator
- [ ] 83. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat
- [ ] 84. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/DateTimeFormat_Options
- [ ] 85. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/formatRangeToParts
- [ ] 86. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DisplayNames
- [ ] 87. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat
- [ ] 88. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Locale
- [ ] 89. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
- [ ] 90. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules
- [ ] 91. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat
- [ ] 92. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator
- [ ] 93. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON
- [ ] 94. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
- [ ] 95. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
- [ ] 96. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number
- [ ] 97. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty/Additional_examples
- [ ] 98. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
- [ ] 99. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy
- [ ] 100. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler
- [ ] 101. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect
- [ ] 102. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/Comparing_Reflect_and_Object_methods
- [ ] 103. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
- [ ] 104. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes
- [ ] 105. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
- [ ] 106. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol
- [ ] 107. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray
- [ ] 108. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap
- [ ] 109. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet
- [ ] 110. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly
- [ ] 111. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI
- [ ] 112. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent
- [ ] 113. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI
- [ ] 114. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
- [ ] 115. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape
- [ ] 116. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval
- [ ] 117. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite
- [ ] 118. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null
- [ ] 119. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat
- [ ] 120. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
- [ ] 121. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined
- [ ] 122. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape
- [ ] 123. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/uneval
- [ ] 124. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements
- [ ] 125. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/Legacy_generator_function
- [ ] 126. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
- [ ] 127. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block
- [ ] 128. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/break
- [ ] 129. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/class
- [ ] 130. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
- [ ] 131. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/continue
- [ ] 132. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
- [ ] 133. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default
- [ ] 134. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while
- [ ] 135. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/Empty
- [ ] 136. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
- [ ] 137. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for
- [ ] 138. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of
- [ ] 139. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for_each...in
- [ ] 140. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in
- [ ] 141. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
- [ ] 142. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function
- [ ] 143. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*
- [ ] 144. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else
- [ ] 145. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import
- [ ] 146. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta
- [ ] 147. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label
- [ ] 148. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let
- [ ] 149. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return
- [ ] 150. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch
- [ ] 151. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw
- [ ] 152. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var
- [ ] 153. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while
- [ ] 154. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/with
- [ ] 155. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode
- [ ] 156. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode/Transitioning_to_strict_mode
- [ ] 157. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas
- [ ] 158. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Shells
- [ ] 159. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/JavaScript_technologies_overview
- [ ] 160. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management
- [ ] 161. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions
- [ ] 162. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/at-cc-on
- [ ] 163. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/at-if
- [ ] 164. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/at-set
- [ ] 165. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/ActiveXObject
- [ ] 166. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Debug
- [ ] 167. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Enumerator
- [ ] 168. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/GetObject
- [ ] 169. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript
- [ ] 170. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_2015_support_in_Mozilla
- [ ] 171. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_5_support_in_Mozilla
- [ ] 172. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.1
- [ ] 173. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.2
- [ ] 174. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.3
- [ ] 175. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.4
- [ ] 176. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.5
- [ ] 177. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.6
- [ ] 178. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.7
- [ ] 179. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.8
- [ ] 180. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.8.1
- [ ] 181. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.8.5
- [ ] 182. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/The_performance_hazards_of__%5B%5BPrototype%5D%5D_mutation

**Comment by ddbeck** [comment]
@Elchi3 Thank you so much for doing this! 💯

**Comment by wbamberg** [comment]
I've fixed the "file-require-recipe-unique" ones. They are mostly because of error pages like [`ReferenceError()`](https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError/ReferenceError) containing an "Error" tag. I wasn't sure whether we ought to remove them or choose something different to flag "javascript-error" recipes. But I decided to go for it in the end. But let me know if you think this was a bad choice, we can always reinstate.

**Comment by Elchi3** [comment]
I've tagged many more pages. The "funny" thing is that I'm now discovering pages I've never seen before. :neutral_face:

**Comment by Elchi3** [comment]
Some random thoughts for the remaining pages. Would love some feedback, so I feel more empowered to take some decisions :)

@chrisdavidmills What was the comment on the Microsoft docs in the PAB meeting? Do we need to start supporting this material in or stumptown efforts?

I'm not sure what to do with the "New in JS" pages. It's historical information. Maybe it should move to the archive?


https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/DateTimeFormat_Options this is brand new. Someone created this. In APIs this would be a dictionary, I guess. I don't think I want to support this content type in JS docs, though. Not a fan of isolated dictionary docs.

If we remove the index page from jsSidebar, then I guess we could delete the index page.

**Comment by ddbeck** [comment]
> I'm not sure what to do with the "New in JS" pages. It's historical information. Maybe it should move to the archive?

I know I'm not a decision maker on this, but I hope we delete or archive this stuff. All but one of those pages concerns changes that landed nearly a decade ago. I imagine BCD or browser release notes are going to be more meaningful for this sort of thing anyway.

**Comment by wbamberg** [comment]
> If we remove the index page from jsSidebar, then I guess we could delete the index page.

I thought DocStatus used the index page: https://github.com/mdn/kumascript/blob/master/macros/DocStatus.ejs#L44 ? For the same reason I've been thinking that we can't remove the `{{Index}}` macro: https://github.com/mdn/sprints/issues/2524#issuecomment-593704900.

**Comment by ddbeck** [comment]
I've taken notes on the pages that still need handling. There's some new stuff here, but there's a little bit of recap too. I've tried to highlight new proposals that I'm making.

### 5. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Differential_inheritance_in_JavaScript
  
**Proposal**: Delete this page.

I can't figure out where this `inherit()` method comes from or I would've suggested moving to the method page's examples. But the sample code doesn't run anyway, so I say dump the page outright.

### 8. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Index

Discussed above. Seems like we're stuck with this one, for now. No action.

### 84. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/DateTimeFormat_Options

Discussed above.

**Proposal**: fold this content into the parent `DateTimeFormat` page.

### 97. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty/Additional_examples

**Proposal**: fold into the examples section of `defineProperty`.

### 102. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/Comparing_Reflect_and_Object_methods

I've tagged this as a guide page.

**Proposal**: correct me if I'm wrong. 😄

### 104. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes

I guess this is a guide page, but if we ignore it long enough, it'll become historical and irrelevant. 🙈 Open to ideas on this one.

### 118. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null
### 121. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined

I'm tagging these as language features, since these hew closely to the recipes already.

**Proposal**: correct me if I'm wrong. 😄

### 161. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions

And it's children. Discussed above. We're waiting to hear from the PAB before we can act on these.

### 169. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript

And it's children. Discussed above. I think we need @chrisdavidmills for consensus on archiving or deleting.

### 182. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/The_performance_hazards_of__%5B%5BPrototype%5D%5D_mutation

This is a draft page, and one that hasn't been edited in nearly a year. I'd say that's abandoned, really. But I'm wondering if we ought to detect draft pages in the linter and demote all messages about them to warnings.

**Proposal**: Ignore or delete.

**Comment by wbamberg** [comment]
One thing to consider is how far we want to fix the content for its own sake, and how far we just want to make it scrapable, and worry about fixing it afterwards. For example, in some of these cases (like the "New in JS") the simplest thing is just tag them as guide pages. That avoids having conversations about "is this content really deletable". Treating pages as guide pages only becomes impossible when that have special custom content (like interactive examples) that guide pages don't support.

I'm not exactly saying we should do the minimum needed to make the content scrapable, but I am worried that once you start down the road of "fix the content on the way", then there's so much extra work in there, and I'm not sure where you stop.

> 118. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null
> 121. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined
>
> I'm tagging these as language features, since these hew closely to the recipes already.
>
> Proposal: correct me if I'm wrong. 😄

That sounds plausible to me. From what I can tell from the [spec](https://tc39.es/ecma262/#sec-ecmascript-data-types-and-values), they are the only values of the `Null` and `Undefined` types. So they're not "global objects". Should we also then move them out of "Global_Objects"? For things like the sidebar, we are implicitly assuming that all the things in /Global_Objects are actually "javascript-class" pages.

> I'm wondering if we ought to detect draft pages in the linter and demote all messages about them to warnings.

I'm not sure about this. For one thing [we are also planning to remove `{{Draft}}`](https://github.com/mdn/sprints/issues/2524#issuecomment-593704900). For another thing though: draft pages that are unscrapable still have to be dealt with, either by deleting them or fixing them.

**Comment by chrisdavidmills** [comment]
> I'm not sure about this. For one thing we are also planning to remove {{Draft}}. For another thing though: draft pages that are unscrapable still have to be dealt with, either by deleting them or fixing them.

One thing I proposed as part of my [localization plan](https://docs.google.com/document/d/1TAXMMq6K_gswHdc19zzI_ihXNrNeCROgJNY_zef3ZoQ/edit#heading=h.pomnf9v0vspr) (the part about reducing the amount of content) was simply scrapping anything in draft that had bene in draft for any significant length of time, unless it really looks like content is valuable, in which case we should sort it out (maybe file a sprint P1 bug for it?)

**Comment by ddbeck** [comment]
> One thing to consider is how far we want to fix the content for its own sake, and how far we just want to make it scrapable, and worry about fixing it afterwards.

This is a good point, but I'm still supportive of deletions, at least. It's less content to scrape and odds are good that it improves the corpus as a whole. I agree that fixes like combining pages is not essential to the overall scraping effort.

Items like 97. https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty/Additional_examples are complicated, though. If they were part of a non-guide page, then we could scrape them as examples. As a guide page, they're just dumb code blocks.

**Comment by Elchi3** [comment]
> I thought DocStatus used the index page: https://github.com/mdn/kumascript/blob/master/macros/DocStatus.ejs#L44 ? For the same reason I've been thinking that we can't remove the {{Index}} macro: #2524 (comment).

I forgot about this. Thanks. Well, I think we could remove it from doc status and from the sidebar and then delete it. I think doc status pages shouldn't make it to github anyway.

>  https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Differential_inheritance_in_JavaScript Proposal: Delete this page.

Deleted.

> https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/DateTimeFormat_Options Proposal: fold this content into the parent DateTimeFormat page.

Done. Redirected to constructor page which documents this.

>  https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty/Additional_examples Proposal: fold into the examples section of defineProperty.

Deleted. These examples were seriously old and the binary flag implementation of the method is like these little libraries that shouldn't be hosted on MDN at all.

> https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer/Planned_changes I guess this is a guide page, but if we ignore it long enough, it'll become historical and irrelevant. see_no_evil Open to ideas on this one.

Yeah, Guide page now. This will be old in a year and then we can archive it.

> https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/The_performance_hazards_of__%5B%5BPrototype%5D%5D_mutation This is a draft page, and one that hasn't been edited in nearly a year. I'd say that's abandoned, really. But I'm wondering if we ought to detect draft pages in the linter and demote all messages about them to warnings. Proposal: Ignore or delete.

Marked as Guide page for now. I searched for this and it seems to be relevant on stackoverflow and other sites, so we should finish this page one day.

> null / undefined
> Should we also then move them out of "Global_Objects"? For things like the sidebar, we are implicitly assuming that all the things in /Global_Objects are actually "javascript-class" pages.

Yeah, probably. I think this needs some sorting, but I assume we should split this out of this issue. Too much work to get into that now.

**Comment by Elchi3** [comment]
I think what is left here (or to be done as follow-ups) is:

- remove index from the sidebar and doc status, then delete index page
- remove "new in js" from sidebar, then move the doc tree to the archive
- Get a word from Microsoft about their pages, then decide what to do with these.

Other follow-ups: Some pages like `null` and `undefined` should probably be moved around (also the Intl namespace, we have an issue for this). But I assume that work is out of scope for the moment, it will come back when we fix pages to comply with their recipes. Does that make sense?

**Comment by ddbeck** [comment]
@Elchi3 Thank you! I think opening follow up issues for these tasks and resolving this user story is the way to go. I'm not exactly quite sure how to write them or organize them though. Are they user stories of their own? What's the categorization of minor tasks like this?

**Comment by wbamberg** [comment]
>  Are they user stories of their own? What's the categorization of minor tasks like this?

Not sure. I just filed https://github.com/mdn/kumascript/pull/1340 to update the sidebar, after that the page updates should be really quick.

For the MS docs, we are dependent on outside teams, so we should file a separate issue I think, and not make it block this one.

**Comment by wbamberg** [comment]
>  Are they user stories of their own? What's the categorization of minor tasks like this?

I just filed https://github.com/mdn/kumascript/pull/1340 to update the sidebar.

I would suggest we file another story for the MS docs. If we wanted to, we could just tag those docs, for the sake of completing this story. I mean, it's fairly obvious what https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/at-if should be tagged as. It'll fail the linter, of course, but that's separate to this story.

I think "remove index from... doc status" is tricky. `DocStatus` is very dependent on `Index`, so much so that removing `Index` essentially means retiring `DocStatus` for the JS pages.

> I think doc status pages shouldn't make it to github anyway.

I agree with this, but I thought doc status was still an important tool for localizers, while we are in Kuma.

**Comment by ddbeck** [comment]
> If we wanted to, we could just tag those docs, for the sake of completing this story. I mean, it's fairly obvious

This is a good point. I've gone ahead and done this. That leaves only `Index` and the "New in JS" pages untagged. I supposed the "New" pages could also be tagged as guides, but it seems like the path to archiving them is pretty straight forward.

**Comment by Elchi3** [comment]
Awesome, I think we might get this done in time then. I agree on the MS docs tagging. It would be nice to hear if we need to further worry about these or not, though. Let's hope we'll get an answer. 

Thanks for the sidebar PR, I have a question on that one, but good to go. If this is deployed, I will move the "New in JS" pages to the archive.

I'm tagging the Index page as "Guide", because Will is probably right, that it needs to stay for the moment.

**Comment by Elchi3** [comment]
Attempt to remove index from doc status https://github.com/mdn/kumascript/pull/1341

**Comment by wbamberg** [comment]
The JsSidebar updates from https://github.com/mdn/kumascript/pull/1340 are merged, deployed, and Ryan has rendered the pages: https://github.com/mdn/kuma/issues/6677

The changes to DocStatus are merged and deployed: https://wiki.developer.mozilla.org/en-US/docs/MDN/Doc_status/JavaScript

So I think we can archive "New in..." and delete the Index page, and then close this?

**Comment by ddbeck** [comment]
> So I think we can archive "New in..." and delete the Index page, and then close this?

This is correct, though it'd probably be a good idea to open an issue for deciding on what to do with the Microsoft Extensions pages (though I'm not exactly sure what the issue should say — it's hard to formulate as a user story). That's not a blocker to completing _this_ user story, but it seems like a good idea to track that work somewhere.

**Comment by ddbeck** [comment]
Also, I would archive the "New in…" pages, but I don't know how to do that. A pointer to instructions for that would be appreciated (assuming it's less work to tell me how to do it than to just do it).

**Comment by chrisdavidmills** [comment]
> what to do with the Microsoft Extensions pages

As an FYI - I asked the PAB crowd what they think about removing the MS-specific stuff, and they generally said it sounded reasonable, although we should probably look at the kind of usage figures the pages have, and see if anyone is going to miss them.

Funnily enough, we got no response at all from the MS reps.

I had a look at the usage figures on GA, and a few of the pages (like ActiveXObject) have a few thousand views per month. But most have very little. I guess the problem here is that it'd be a lot of work to get them updated to start passing linting against the page recipes, for very little gain.

What I think we should do is treat them in the same way as Mozilla/  and Archive/ pages - don't bother to fix them up or aim to get them into structured data, just dump them in the archive so they still exist if someone needs them.

**Comment by Elchi3** [comment]
> So I think we can archive "New in..." and delete the Index page, and then close this?

I moved the "new in"  pages and I deleted the index page.

> What I think we should do is treat them in the same way as Mozilla/ and Archive/ pages - don't bother to fix them up or aim to get them into structured data, just dump them in the archive so they still exist if someone needs them.

I agree wit this and would go forward with moving them to archive if that has consensus with Will and Daniel.

**Comment by ddbeck** [comment]
@chrisdavidmills thank you for asking the PAB!

@Elchi3 Moving to the archive works for me. 👍 And thanks for moving the "New in…" pages.

**Comment by wbamberg** [comment]
I just ran the linter and got:

```
Summary
Macro: jsxref                                                                 7393 warnings  jsxref                                                                 
Macro: JSRef                                                                  677 warnings   jsref                                                                  
Macro: JsSidebar                                                              233 warnings   jssidebar                                                              
Macro: optional_inline                                                        200 warnings   optional_inline                                                        
Macro: Glossary                                                               151 warnings   glossary                                                               
Missing from javascript-method: prose.description                             143 errors     javascript-method/prose.description                                    
Linting data.info_box ingredient is unimplemented                             110 warnings   javascript-property/data.info_box                                      
Missing from javascript-error: prose.short_description                        73 errors      javascript-error/prose.short_description                               
Missing from javascript-constructor: prose.description                        63 errors      javascript-constructor/prose.description                               
Macro: ReadOnlyInline                                                         61 warnings    readonlyinline                                                         
Missing from javascript-method: data.specifications                           50 errors      javascript-method/data.specifications                                  
Macro: js_property_attributes                                                 49 warnings    js_property_attributes                                                 
Missing from javascript-method: data.examples                                 49 errors      javascript-method/data.examples                                        
Macro: bug                                                                    43 warnings    bug                                                                    
Macro: Non-standard_Header                                                    43 warnings    non-standard_header                                                    
Linting data.class_constructor ingredient is unimplemented                    35 warnings    javascript-class/data.class_constructor                                
Macro: PreviousNext                                                           32 warnings    previousnext                                                           
Expected SpecName macro for javascript-method: data.specifications            28 errors      javascript-method/data.specifications/expected-macro                   
Macro: Obsolete_Header                                                        26 warnings    obsolete_header                                                        
Missing from javascript-language-feature: data.examples                       24 errors      javascript-language-feature/data.examples                              
Missing from javascript-property: data.specifications                         24 errors      javascript-property/data.specifications                                
Macro: deprecated_header                                                      23 warnings    deprecated_header                                                      
Missing from javascript-method: data.browser_compatibility                    23 errors      javascript-method/data.browser_compatibility                           
Missing from javascript-property: prose.description                           22 errors      javascript-property/prose.description                                  
Expected h2#Specifications                                                    22 errors      javascript-method/data.specifications/expected-heading                 
Expected h2#Browser_compatibility                                             22 errors      javascript-method/data.browser_compatibility/expected-heading          
Missing from javascript-constructor: prose.syntax                             20 errors      javascript-constructor/prose.syntax                                    
Missing from javascript-language-feature: prose.description                   19 errors      javascript-language-feature/prose.description                          
Missing from javascript-constructor: data.examples                            19 errors      javascript-constructor/data.examples                                   
Expected SpecName macro for javascript-property: data.specifications          17 errors      javascript-property/data.specifications/expected-macro                 
Missing from javascript-property: data.examples                               17 errors      javascript-property/data.examples                                      
Missing from javascript-language-feature: data.specifications                 14 errors      javascript-language-feature/data.specifications                        
Missing from javascript-class: data.examples                                  11 errors      javascript-class/data.examples                                         
Macro: EmbedTest262ReportResultsTable                                         10 warnings    embedtest262reportresultstable                                         
Missing from javascript-error: prose.message                                  9 errors       javascript-error/prose.message                                         
Missing from javascript-error: prose.error_type                               9 errors       javascript-error/prose.error_type                                      
Missing from javascript-error: prose.what_went_wrong                          9 errors       javascript-error/prose.what_went_wrong                                 
Missing from javascript-method: prose.see_also                                9 errors       javascript-method/prose.see_also                                       
Macro: DOMxRef                                                                8 warnings     domxref                                                                
Missing from javascript-language-feature: prose.syntax                        8 errors       javascript-language-feature/prose.syntax                               
Missing from javascript-property: prose.see_also                              8 errors       javascript-property/prose.see_also                                     
Linting prose.overview ingredient is unimplemented                            7 warnings     landing-page/prose.overview                                            
Linting data.link_lists ingredient is unimplemented                           7 warnings     landing-page/data.link_lists                                           
Expected SpecName macro for javascript-language-feature: data.specifications  7 errors       javascript-language-feature/data.specifications/expected-macro         
Expected h2#Specifications                                                    7 errors       javascript-language-feature/data.specifications/expected-heading       
Missing from javascript-class: data.specifications                            7 errors       javascript-class/data.specifications                                   
Expected h2#Specifications                                                    7 errors       javascript-property/data.specifications/expected-heading               
Missing from javascript-namespace: data.examples                              6 errors       javascript-namespace/data.examples                                     
Expected h2#Browser_compatibility                                             6 errors       javascript-property/data.browser_compatibility/expected-heading        
Missing from javascript-property: data.browser_compatibility                  6 errors       javascript-property/data.browser_compatibility                         
Missing from javascript-language-feature: data.browser_compatibility          5 errors       javascript-language-feature/data.browser_compatibility                 
Expected h2#Browser_compatibility                                             4 errors       javascript-language-feature/data.browser_compatibility/expected-heading
Macro: Draft                                                                  4 warnings     draft                                                                  
Expected SpecName macro for javascript-class: data.specifications             4 errors       javascript-class/data.specifications/expected-macro                    
Missing from javascript-class: data.browser_compatibility                     4 errors       javascript-class/data.browser_compatibility                            
Macro: EmbedLiveSample                                                        3 warnings     embedlivesample                                                        
Missing from javascript-language-feature: prose.see_also                      3 errors       javascript-language-feature/prose.see_also                             
Macro: SeeCompatTable                                                         3 warnings     seecompattable                                                         
Expected h2#Specifications                                                    3 errors       javascript-class/data.specifications/expected-heading                  
Expected h2#Browser_compatibility                                             3 errors       javascript-class/data.browser_compatibility/expected-heading           
Macro: Previous                                                               2 warnings     previous                                                               
Missing from javascript-class: prose.see_also                                 2 errors       javascript-class/prose.see_also                                        
Expected SpecName macro for javascript-constructor: data.specifications       2 errors       javascript-constructor/data.specifications/expected-macro              
Missing from javascript-constructor: data.specifications                      2 errors       javascript-constructor/data.specifications                             
Macro: Next                                                                   1 warning      next                                                                   
Macro: ListSubPages                                                           1 warning      listsubpages                                                           
Missing from javascript-constructor: prose.see_also                           1 error        javascript-constructor/prose.see_also                                  
Missing from javascript-error: data.examples                                  1 error        javascript-error/data.examples                                         
Missing from javascript-method: prose.short_description                       1 error        javascript-method/prose.short_description                              
Missing from javascript-constructor: prose.short_description                  1 error        javascript-constructor/prose.short_description                         
Expected h2#Specifications                                                    1 error        javascript-namespace/data.specifications/expected-heading              
Missing from javascript-namespace: data.specifications                        1 error        javascript-namespace/data.specifications                               
Expected h2#Browser_compatibility                                             1 error        javascript-namespace/data.browser_compatibility/expected-heading       
Missing from javascript-namespace: data.browser_compatibility                 1 error        javascript-namespace/data.browser_compatibility                        
Missing from javascript-method: prose.syntax                                  1 error        javascript-method/prose.syntax                                         
Missing from javascript-property: prose.short_description                     1 error        javascript-property/prose.short_description                            
```

-> 0 recipe errors, so I think we can close this issue.

**Comment by wbamberg** [comment]
> > What I think we should do is treat them in the same way as Mozilla/ and Archive/ pages - don't bother to fix them up or aim to get them into structured data, just dump them in the archive so they still exist if someone needs them.
> 
> I agree with this and would go forward with moving them to archive if that has consensus with Will and Daniel.

I'm fine with this, but I don't really have any particular authority to decide what to do with these pages. An alternative, if we want to keep them in /JavaScript, is to tag them as guide pages, which I think would work OK (they don't contain any exciting macros we'd need to worry about, although they do use things like `<dl>` which would be a little, um. *modified* in the transition).

**Comment by chrisdavidmills** [comment]
> An alternative, if we want to keep them in /JavaScript, is to tag them as guide pages, which I think would work OK

Wouldn't that be a bit of a hack though? It seems wrong to start hacking stumptown this early on in the process ;-) I like the idea of doing it properly, or not at all. If we were to put them in the archive, they would still be available, and redirects would make sure that any links in place won't break?

**Comment by ddbeck** [comment]
> Wouldn't that be a bit of a hack though? It seems wrong to start hacking stumptown this early on in the process

We talked a little about this risk when we were in London. It's pretty unrealistic to expect that we'd be able to clean up and structure everything coming into Stumptown. The guide pages are the shallowest of shallow structure, so I don't think this is a hack, so much as admission that we're going to have to drag some legacy content along or rely on the community to help us do some cleanup _in_ Stumptown.

**Comment by chrisdavidmills** [comment]
> I don't think this is a hack, so much as admission that we're going to have to drag some legacy content along or rely on the community to help us do some cleanup in Stumptown.

OK, WFM. Thanks for the explanation!
